actions/_config.yaml

```
@@ -10,8 +10,8 @@ target_repository:
   checkout: master
   # OPTIONAL, define installing package extras
   install_extras: all
-  # OPTIONAL, install additional requirements
-  # install_file: requirements.txt
+  # OPTIONAL, install additional requirements from a file
+  requirement_files: requirements.txt
```

Would maybe do `requirements_files` instead:
```suggestion
  requirements_files: requirements.txt
```